### ISSUE TITLE
Select / Option visibility wrongly calculated for very wide options

### ISSUE THREAD

**Issue by CloCkWeRX** [jnicklas/capybara#1655]
We are using ~> 2.6.0 and started receiving a visibility related regression.

We're doing:
```
e.first("select option", text: "General - Private Clients").select_option
```
but get warnings the element next to it would receive the click
```
Selenium::WebDriver::Error::UnknownError: Element is not clickable at point (656.3666687011719, 69.16667175292969). Other element would receive the click: <select tabindex="22" class="ember-view ember-select selector" id="ember1875"></select>
```

It turns out this is because of an extra long option causing the displayed options to be very wide; and Capybara thinks that the non-displayed options would overlap, maybe due to z-index?, with the selected option.

![would_receive_click](https://cloud.githubusercontent.com/assets/365751/13166440/d4aac184-d717-11e5-8cef-f128958c5edf.png)

**Comment by twalpole** [comment]
Since all capybara does for select_option when using selenium is tell selenium to click the option unless it's already selected -- this would seem to be a selenium issue.  If you're not already using the latest selenium-webdriver try updating - If you are you may want to report an issue on the selenium project.

As a side note you could write your code as

    e.first(:option, "General - Private Clients").select_option

it's not a great benefit when dealing with option elements since the :option selector doesn't add any extra filters, but getting in the habit of using the provided [selectors](https://github.com/jnicklas/capybara/blob/master/lib/capybara/selector.rb#L279) can lead to better reading tests with more flexibility

**Comment by twalpole** [comment]
Looks like https://github.com/SeleniumHQ/selenium/commit/b21a4f52d37588d2938aace2f47c93d442cd052e is probably the fix for your issue - and was released in selenium-webdriver 2.52.0
